history-of-quakers/modernized/06-chapter-4.adoc

```diff
@@ -426,8 +452,13 @@ and the Lord never sent them.`'
 
 A good while after this, the magistrates gave out the following warrant:
 
+[.embedded-content-document.legal]
+--
+
+[.signed-section-context-open]
 City of Bristol.
 
+[.letter-heading]
```

Comment:
hmmm... we never really do a `letter-heading` _after_ a context-open.  I see why you did it though, it's sort of weird.  Let's change this to a 'salutation' instead, that's my best guess of how to handle it.

Reply:
okay, I did wonder about that, but it seemed so much like a letter heading! I was thinking about 75% rule. 😄